components/CustomPage.js

```diff
@@ -0,0 +1,14 @@
+import Navbar from "./Navbar";
+import styles from "../styles/Home.module.css";
+const Index = ({ children }) => {
+  return (
+    <div>
+      <div className={styles.container}>
+        <Navbar />
+      </div>
+      <div className={styles.container}>{children}</div>
+    </div>
+  );
+};
+
+export default Index;
```

Comment:
I think we can have this code in _app.js?

Reply:
I think we also should name the component as `Layout`